website/docs/community/design-philosophy.md

```diff
@@ -0,0 +1,89 @@
+---
+id: design-philosophy
+title: Velox Design Philosophy
+---
+
+## Summary
+
+This page lists a set of directional principles and values meant to guide
+contributors and maintainers as they develop the Velox project. These are not
+meant to be hard-and-fast rules, but to inform decision making and help guide
+discussions that may come up during the development of Velox. 
+
+## Velox Mission
+
+The Velox project aspires to:
+
+> “Accelerate data management systems by providing a **reliable** and
+> **unified** **state-of-the-art** **open-source** execution platform”
+
+The *Velox community* is composed of a set of highly-skilled individuals with the
+shared purpose of achieving this mission through the development and
+maintenance of the **Velox open-source library** and its ecosystem. In order to
+support and inform community efforts as we progress towards achieving the
+mission, a few overarching principles and values are highlighted below.
+
+## Velox Principles
+
+* **Reliable.** To become the true execution underpinnings for data management
+  systems across the industry, Velox strives for excellence in reliability.
+  This means that as a community we heavily invest in enhancing our test
+  infrastructure, and building sophisticated database testing and validation
+  frameworks. It means that during Velox development we do not compromise on
+  stability, reliability, or correctness.
+
+* **Unified.** We envision that by unifying execution across data management
+  systems (hence reducing the existing fragmentation), we can be more efficient
+  as an engineering community. Therefore, Velox aspires to provide components
+  which are engine- and dialect-agnostic, containing extensibility APIs through
+  which the behavior of the library can be extended to match a desired
+  semantic.
+
+* **State-of-the-art.** As a unified platform, we believe in the opportunity of
+  centralizing advanced optimizations and techniques previously only available
+  in individual monolithic systems (many of which proprietary), making them
+  easily available everywhere. Although there is a large set of optimizations
+  that can be added (each adding a layer of complexity), we prioritize adding
+  the features that provide the largest performance improvement opportunity.
```

Comment:
We used to have some wording that clarifying that we prioritize real-world scenarios over the benchmarks. Looks like this might got lost somehow.

Reply:
@mbasmanova it's mentioned in the Velox Values section under data-driven. Is that what you are looking for?